Unit tests failing with sacrebleu 1.4.2

The latest version on pypi (1.4.2) has 15 unit tests failing. Most of them for this reason:
```
    @pytest.mark.parametrize("hypotheses, references, expected_score", test_cases_keep_whitespace)
    def test_chrf_keep_whitespace(hypotheses, references, expected_score):
        score = sacrebleu.corpus_chrf(hypotheses, references, 6, 3, remove_whitespace=False)
>       assert abs(score - expected_score) < EPSILON
E       TypeError: unsupported operand type(s) for -: 'CHRF' and 'float'
```

--- a/test/test_chrf.py
+++ b/test/test_chrf.py
@@ -38,10 +38,10 @@ test_cases_keep_whitespace = [
 
 @pytest.mark.parametrize("hypotheses, references, expected_score", test_cases)
 def test_chrf(hypotheses, references, expected_score):
-    score = sacrebleu.corpus_chrf(hypotheses, references, 6, 3)
+    score = sacrebleu.corpus_chrf(hypotheses, references, 6, 3).score
     assert abs(score - expected_score) < EPSILON
 
 @pytest.mark.parametrize("hypotheses, references, expected_score", test_cases_keep_whitespace)
 def test_chrf_keep_whitespace(hypotheses, references, expected_score):
-    score = sacrebleu.corpus_chrf(hypotheses, references, 6, 3, remove_whitespace=False)
+    score = sacrebleu.corpus_chrf(hypotheses, references, 6, 3, remove_whitespace=False).score
     assert abs(score - expected_score) < EPSILON

